integration-tests/mock-components/frozen-counter.js

```diff
@@ -0,0 +1,20 @@
+const { registerHtml, useObservable } = require('../../src/tram-one')
+const html = registerHtml()
+
+module.exports = () => {
+	const [,, rawCounter] = useObservable({ now: { clicks: 0 } })
+	const onclick = () => {
+		// while a spread operator would work for single nested objects
+		// for objects that are deeply nested, we need to use raw to
+		// transform the entire object to it's raw value
+		rawCounter.now.clicks += 1
+	}
```

Comment:
Basically what this comment says... In reality, this may be rarely used, but for beer-cart.io, it was required to pass and interact with an object, and then wholesale commit to changing it or not.

It appears the that having observables change the UI on mutation can be a double edged sword.